### PR TITLE
ArtPaint: Updated tool setup dialogs to fix layout issues

### LINKED ISSUE #9: Air brush tool labels are cut off
**Creator:**
**URL: http://dev.osdrawer.net/issues/592**

_CAUTION: comments not extracted from issues. See OsDrawer or the WaybackMachine for issue comments._

### LINKED ISSUE #10: Blur tool labels are cut off
**Creator:**
**URL: http://dev.osdrawer.net/issues/593**

_CAUTION: comments not extracted from issues. See OsDrawer or the WaybackMachine for issue comments._

### LINKED ISSUE #11: Brush tool labels and controls are cut off
**Creator:**
**URL: http://dev.osdrawer.net/issues/594**

_CAUTION: comments not extracted from issues. See OsDrawer or the WaybackMachine for issue comments._

### LINKED ISSUE #12: Color selector tool label cut off
**Creator:**
**URL: http://dev.osdrawer.net/issues/595**

_CAUTION: comments not extracted from issues. See OsDrawer or the WaybackMachine for issue comments._

### LINKED ISSUE #13: Eraser tool label and slider cut off
**Creator:**
**URL: http://dev.osdrawer.net/issues/596**

_CAUTION: comments not extracted from issues. See OsDrawer or the WaybackMachine for issue comments._

### LINKED ISSUE #14: Free line tool label cut off
**Creator:**
**URL: http://dev.osdrawer.net/issues/597**

_CAUTION: comments not extracted from issues. See OsDrawer or the WaybackMachine for issue comments._

### LINKED ISSUE #15: Hairy brush tool labels cut off
**Creator:**
**URL: http://dev.osdrawer.net/issues/598**

_CAUTION: comments not extracted from issues. See OsDrawer or the WaybackMachine for issue comments._

### LINKED ISSUE #16: Selector tool label and slider cut off
**Creator:**
**URL: http://dev.osdrawer.net/issues/599**

_CAUTION: comments not extracted from issues. See OsDrawer or the WaybackMachine for issue comments._

### LINKED ISSUE #17: Straight line tool label and slider cut off
**Creator:**
**URL: http://dev.osdrawer.net/issues/600**

_CAUTION: comments not extracted from issues. See OsDrawer or the WaybackMachine for issue comments._

### LINKED ISSUE #19: Transparency tool labels cut off
**Creator:**
**URL: http://dev.osdrawer.net/issues/602**

_CAUTION: comments not extracted from issues. See OsDrawer or the WaybackMachine for issue comments._

### PR DESCRIPTION
**Pull request by dsizzle**
Minor updates to most of the tools setup dialogs to fix layout
issues.  Didn't fix the Brush tool as that change is a bit more
complex and will be a separate commit.

Fixes #9, fixes #10, fixes #11, fixes #12, fixes #13, fixes #14, fixes #15, fixes #16, fixes #17, fixes #19

### CODE REVIEW

**Comment by korli** [comment]
@humdingerb any chance you have a look?

**Comment by humdingerb** [comment]
Had to re-activate an old 32bit beta3 installation and update to cuurent nightly, because it won't build under beta3 anymore since that cpu-speed/frequency/whatever change. :)

I compared to the versioncurrently in HaikuDepot and have the following observations:
+ Hairy brush: Size/Hairs slider doesn't work
+ Air brush: Size/Flow slider doesn't work
+ Blur tool: Size slider doesn't work
+ Transparency tool: Size slider doesn't work
+ Selector tool: Tolerance slider doesn't work

Left-align checkboxes for
+ Straight line
+ Blur tool
+ Fill tool

Use same spacing/padding for all widgets, esp. checkboxes and radio buttons. IMO the spacing of the "Mode" checkboxes of the Fill tool setup are perfect. (Though that panel having to "Mode" sections isn't nice...) Probably best to use spacing constants everywhere.

File: artpaint/application/FrenchStrings.h [hunk 1]
```diff
@@ -155,6 +155,7 @@
 #define	FRENCH_OPEN_PROJECT_HELP_STRING		"Ouvre un projet du disque."
 #define	FRENCH_OPEN_PROJECT_STRING				"Ouvrir un projet…"
 #define	FRENCH_OPEN_STRING						"Ouvrir"
+#define	FRENCH_OPTIONS_STRING					"Choix"
```
Comment (korli) [inline artpaint/application/FrenchStrings.h:158]:
```suggestion
#define	FRENCH_OPTIONS_STRING					"Préférences"
```

**Comment by korli** [comment]
> Had to re-activate an old 32bit beta3 installation and update to cuurent nightly, because it won't build under beta3 anymore since that cpu-speed/frequency/whatever change. :)

Hmm, I thought it came before beta3, will check.

**Comment by humdingerb** [comment]
@korli, dunno what went wrong before, but ArtPaint now does build on 64bit... Nice I don't have to boot into 32bit any more.

**Comment by humdingerb** [comment]
Nicely done, @dsizzle ! Can we merge?

**Comment by korli** [comment]
@humdingerb What's with your feedback from 20th March?

**Comment by dsizzle** [comment]
@humdingerb yes, this is ready to merge if it looks good to you.

with respect to building, the whole purpose of the system_info patch was because I couldn't build ArtPaint at all and figured I'd modernize it a bit in the process. I am using a nightly and so I did try beta3 and it seems like the cpu frequency changes came after beta3. I can remove the 'fallback' code which is where it fails if that's desired for beta3 compatibility.

**Comment by korli** [comment]
> with respect to building, the whole purpose of the system_info patch was because I couldn't build ArtPaint at all and figured I'd modernize it a bit in the process. I am using a nightly and so I did try beta3 and it seems like the cpu frequency changes came after beta3. I can remove the 'fallback' code which is where it fails if that's desired for beta3 compatibility.

I already removed this code in e7ead3219636c53a00eb55ab0372bdb4b99028ec

**Comment by humdingerb** [comment]
> What's with your feedback from 20th March?

All been resolved nicely by dsizzle. Let's merge!

**Pull request merged**